presto-docs/src/main/sphinx/release/release-0.214.rst

```diff
@@ -0,0 +1,51 @@
+=============
+Release 0.214
+=============
+
+General Changes
+---------------
+
+* Add feature and session toggles for *top-N row number* optimization.
+* Add ``ngrams`` function to generate the N-grams from an array.
+* Add ``qdigest`` type and associated functions.
+* Add statistics calculator support for ``TINYINT``, ``SMALLINT`` and ``VARBINARY`` literals.
```

Comment:
Please explain this in terms of end-user impact. Or is this an internal-only feature that doesn't need to be documented?

Reply:
@sopel39 is there a user-visible change to document here? Reading #11743, it looks like it improve the cost-estimation with scalar and magic and functions.

Reply:
It impacts query planning. Without those changes, we've seen queries greatly misetimated and multiple times more expensive to run. Therefore it is user-visible change.

Reply:
Maybe it should be rephased to:
```
Add CBO support for ``TINYINT``, ``SMALLINT`` and ``VARBINARY`` literals, which improves planning and query times.
```